### DISCUSSION TITLE
I Can't use Copilot Chat in vs code

### DISCUSSION THREAD

**Discussion by weizijun** [community/community#56464] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My vs code version is : 1.78.2 (Universal)
I have installed GitHub Copilot Chat.
But I can't use it.
I'm in.
"The GitHub Copilot chat private beta is already enabled on your account!"

I run the Copilot Chat command, but it return:
command 'github.copilot.ghpr.applySuggestion' not found

**Comment by weizijun** [comment]
![image](https://github.com/community/community/assets/5070449/f8010616-5cdb-4aad-9c8b-186a9325c072)

**Comment by weizijun** [comment]
Version: 1.78.2 (Universal)
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:44:45.204Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin x64 19.5.0
Sandboxed: Yes

**Comment by newjcj** [comment]
我也是遇到了一样的问题

**Reply by newjcj** [reply]
我解决了，哈哈

**Reply by newjcj** [reply]
重装下vscode

**Reply by weizijun** [reply]
原来我装的不是insiders版本，装上insiders版本后可以使用了

**Comment by weizijun** [comment]
My vs code version is not insiders. After install the insiders version, the GitHub Copilot Chat can work

**Reply by mmraja83** [reply]
Nope... I still see the error.
